orchagent/notifications.cpp

```diff
@@ -86,3 +87,4 @@ sai_switch_notification_t switch_notifications
     on_switch_shutdown_request,
     NULL
 };
+*/
```

Comment:
What is the action items here? Do you mean cleaning the commented code?

---
In reply to: [110933718](https://github.com/lguohan/sonic-swss/pull/1#discussion_r110933718) [](ancestors = 110933718)